Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM quay.io/mozmar/base:latest
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PIP_DISABLE_PIP_VERSION_CHECK=1
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y --no-install-recommends python2.7 libpython2.7 python-dev \
+    python-pip gettext build-essential \
+    libxml2-dev libxslt1-dev zlib1g-dev git\
+    libjpeg-dev libffi-dev libssl-dev libxslt1.1\
+    libmysqlclient-dev software-properties-common apt-transport-https
+
+# Install mariadb-client 5.5
+RUN apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 0xcbcb082a1bb943db
+RUn add-apt-repository 'deb [arch=amd64,i386] http://sfo1.mirrors.digitalocean.com/mariadb/repo/5.5/debian wheezy main'
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends  mariadb-client-5.5 libmariadbclient-dev
```

Comment:
Hmm not sure if that's needed here. We probably are going to run on mariadb 10+ so we can just use the default packages. Feel free to leave this here for now and we can revise later